turbo.json

```diff
@@ -1,84 +1,89 @@
 {
-    "$schema": "https://turborepo.org/schema.json",
-    "baseBranch": "origin/trunk",
-    "pipeline": {
-        "lint": {
-            "cache": false
-        },
-        "lint:fix": {
-            "cache": false
-        },
+	"$schema": "https://turborepo.org/schema.json",
+	"baseBranch": "origin/trunk",
+	"pipeline": {
+		"lint": {
+			"cache": false
+		},
+		"lint:fix": {
+			"cache": false
+		},
 
-        "build": {
-            "dependsOn":  [ "^build" ],
-            "inputs": [
-                "src/*.js",
-                "src/**/*.js",
-                "src/*.jsx",
-                "src/**/*.jsx",
-                "src/*.ts",
-                "src/**/*.ts",
-                "src/*.tsx",
-                "src/**/*.tsx",
-                "src/*.php",
-                "src/**/*.php",
-                "includes/*.php",
-                "includes/**/*.php"
-            ],
-            "outputs": [
-                "dist/**",
-                "build/**",
-                "build-module/**",
-                "build-style/**",
-                "build-types/**"
-            ]
-        },
-        "woocommerce#build": {
-            "dependsOn":  [ 
-                "^build",
-                "woocommerce/client/admin#build",
-                "woocommerce/client/legacy#build"
-            ],
-            "outputs": [],
-            "inputs": [
-                "src/*.php",
-                "src/**/*.php",
-                "includes/*.php",
-                "includes/**/*.php"
-            ]
-        },
+		"build": {
+			"dependsOn": [ "^build" ],
+			"inputs": [
+				"src/*.js",
+				"src/**/*.js",
+				"src/*.jsx",
+				"src/**/*.jsx",
+				"src/*.ts",
+				"src/**/*.ts",
+				"src/*.tsx",
+				"src/**/*.tsx",
+				"src/*.php",
+				"src/**/*.php",
+				"includes/*.php",
+				"includes/**/*.php"
+			],
+			"outputs": [
+				"dist/**",
+				"build/**",
+				"build-module/**",
+				"build-style/**",
+				"build-types/**"
+			]
+		},
+		"woocommerce/client/admin#build": {
+			"inputs": [ "js/**/*.js", "js/**/*.scss" ]
```

Comment:
[Turborepo's Documentation](https://turborepo.org/docs/core-concepts/pipelines#implicit-dependencies-and-specific-package-tasks), notes that package tasks don't inherit cache configuration. It specifically calls out `"outputs"` but is this true of `inputs` as well? I _think_ this might be getting rid of the inputs for WooCommerce Admin, right?

Reply:
@ObliviousHarmony ah shoot yeah you're right about that. 🤔